.gitignore

```diff
@@ -7,4 +7,5 @@ build
 !buildSrc/src/main/groovy/org/springframework/restdocs/build/
 target
 .idea
-*.iml
\ No newline at end of file
+*.iml
+.DS_Store
```

Comment:
This is OS-specific so it doesn't belong in the repository's `.gitignore`. It should go in your own [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead.